app/src/main/java/com/pps/globant/fittracker/model/avatars/jsonAvatarClasses/Item.java

```diff
@@ -0,0 +1,32 @@
+
+package com.pps.globant.fittracker.model.avatars.jsonAvatarClasses;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Item {
```

Comment:
Use more representative names please

Reply:
it's a unused component of the Json response from the marvel api, autogenerated just to match the body format of the response and to work with Gson library from OkHttp from retrofit. otherwise doesn't work, but this classes are never used in the app's code. Should i change it anyway?